src/components/ol/Map.vue

```diff
@@ -79,6 +79,14 @@ export default {
       me.setupMapHover();
     }, 200);
   },
+  destroyed () {
```

Comment:
In `MeasureWin` and `OlMeasureController` your are using `destroy`, here you are using `destroyed`. Is this by purpose? Could we harmonize that by taking the same hook / function here and in `MeasureWin` and `OlMeasureController`?

Reply:
`destroyed()` is a [VueJS lifecycle](https://www.digitalocean.com/community/tutorials/vuejs-component-lifecycle)  callback and `destroy()` explicit/user-initiated.

Reply:
Thanks for clarification